pkg/skaffold/deploy/resource/deployment.go

```diff
@@ -165,15 +175,24 @@ func (d *Deployment) ReportSinceLastUpdated() string {
 	for _, p := range d.pods {
 		if s := p.ActionableError().Message; s != "" {
 			result.WriteString(fmt.Sprintf("%s %s %s: %s\n", tab, tabHeader, p, s))
-			for _, l := range p.Logs() {
-				result.WriteString(fmt.Sprintf("%s\n", l))
+			// if, is muted then write container logs to file and last 3 lines to
+			// result.
+			out, writeTrimLines, err := withLogFile(p.Name(), &result, p.Logs(), isMuted)
+			if err != nil {
+				logrus.Debugf("could not create log file %v", err)
```

Comment:
`logrus.Warnf` would seem better here, right?

Reply:
everywhere else in the code, this error is swallowed. My guess is to save screen space. I ended up adding this as debug.